sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala

```diff
@@ -40,6 +40,8 @@ class TextFileFormat extends TextBasedFileFormat with DataSourceRegister {
 
   override def shortName(): String = "text"
 
+  override def toString: String = shortName.toUpperCase
```

Comment:
Yup, I just wanted to say i'd be nicer if they are matched anyway. BTW, it seems it's `ParquetFormat` alone in `ParquetFileFormat` alone whereas its ORC in `OrcFileFormat`.